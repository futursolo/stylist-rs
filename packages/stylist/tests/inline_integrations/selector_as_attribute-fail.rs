fn main() {
    let _ = stylist::css! {
        looks_like_selector(): blue;
    };
}
