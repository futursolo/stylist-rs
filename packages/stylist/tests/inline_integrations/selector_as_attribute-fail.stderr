error: unexpected ';' in selector, did you mean to write an attribute?
 --> $DIR/selector_as_attribute-fail.rs:3:36
  |
3 |         looks_like_selector(): blue;
  |                                    ^
